src/mame/misc/falgas_m89.cpp

```diff
@@ -151,16 +201,52 @@ void falgasm89_state::cbully(machine_config &config)
 	psg.port_b_read_callback().set(FUNC(falgasm89_state::psg_pb_r));
 }
 
+void falgasm89_state::falgasm89(machine_config &config)
+{
+	falgasm89_simple(config);
+
+	I8255(config, "i8255"); // NEC D71055C
+}
+
+void falgasm89_video_state::falgasm89_video(machine_config &config)
+{
+	falgasm89_state::falgasm89(config);
```

Comment:
You don’t need to put the base class name qualifier here – there’s no identically named member in the derived class.

Reply:
Done!